internal/exec/yaml_func_store_test.go

```
@@ -0,0 +1,98 @@
+	})
+	assert.NoError(t, err)
+
+	// Setup test configuration
+	atmosConfig := schema.AtmosConfiguration{
+		Stores: map[string]store.Store{
+			"redis": redisStore,
+		},
+	}
+
+	// Populate the store with some data
+	redisStore.Set("dev", "vpc", "cidr", "10.0.0.0/16")
```

golangci-lint (1.62.2)

Error return value of `redisStore.Set` is not checked

(errcheck)